src/Features/Core/Portable/Completion/Providers/AbstractRecommendationServiceBasedCompletionProvider.cs

```diff
@@ -45,7 +44,7 @@ protected override Task<ImmutableArray<ISymbol>> GetSymbolsWorker(SyntaxContext
                 cancellationToken).ConfigureAwait(false);
 
             // Don't preselect intrinsic type symbols so we can preselect their keywords instead.
-            return symbols.WhereAsArray(s => inferredTypes.Contains(GetSymbolType(s)) && !IsInstrinsic(s));
+            return symbols.Where(s => inferredTypes.Contains(GetSymbolType(s)) && !IsInstrinsic(s)).Select(s => (s, CompletionItemRules.Default)).ToImmutableArray();
```

Comment:
@Pilchie I know, as I said above, my approach to refactoring is doing one thing at a time and improve later as opposed to do everything at the same time, one of the reasons for this is because I'm new to the codebase so trying to do it all in one step can be overwhelming.

I hope it make sense. :)